assets/keymaps/default-macos.json

```diff
@@ -34,7 +34,8 @@
       "alt-shift-enter": "menu::Restart",
       "cmd-shift-w": "workspace::CloseWindow",
       "shift-escape": "workspace::ToggleZoom",
-      "cmd-o": "workspace::Open",
+      "cmd-o": "workspace::OpenFiles",
+      "cmd-k cmd-o": "workspace::Open",
```

Comment:
It seems vscode only has `cmd-o` for open, so this change should not be made here given the default keymap is vscode specific

Reply:
Should I remove the `cmd-k cmd-o` from Windows as well or just the 2 I added?

Reply:
OK done. For future would it have been preferable to amend it? I definitely would have on a lot of projects but I like to ask because I don't want to make people review changes twice if they don't want to no matter how small. Like, as a zed culture thing if I make a tiny PR and have to make a change should I amend if it makes sense, or always make a new commit?